CircleciScripts/pre_start_simulator.sh

```diff
@@ -9,7 +9,7 @@ if [[ -z $test_device_id ]] ; then
   echo "Runtime: '${runtime}'"
 
   # Get the last alphabetical device (probably something in the iPhone X family, as of 2019-05-31)
-  devicetype=$( xcrun simctl list devicetypes iPhone | sort | tail -1 | sed 's/.*(//' | sed 's/).*//' | tr -d '[:space:]' )
+   devicetype=$( xcrun simctl list devicetypes "iPhone " | sort | tail -1 | sed 's/.*(//' | sed 's/).*//' | tr -d '[:space:]' )
```

Comment:
Made this change so that it wont pick other devicetypes like iPad, which will avoid the following error:

```
xcodebuild: error: unreadable input 'runtime:iOS16.2(16.2-20C52)-com.apple.CoreSimulator.SimRuntime.iOS-16-2' at end of value for option 'Destination'

```